Protecs/src/main/java/uk/ac/ucl/protecs/behaviours/Demography.java

```diff
@@ -0,0 +1,224 @@
+package uk.ac.ucl.protecs.behaviours;
+
+import java.util.Arrays;
+import java.util.List;
+
+import sim.engine.SimState;
+import sim.engine.Steppable;
+import uk.ac.ucl.protecs.objects.Household;
+import uk.ac.ucl.protecs.objects.Location;
+import uk.ac.ucl.protecs.objects.Person;
+import uk.ac.ucl.protecs.sim.Params;
+import uk.ac.ucl.protecs.sim.WorldBankCovid19Sim;
+
+public class Demography {
+	
+	public class Aging implements Steppable {
+
+		Person target;
+		int ticksUntilNextBirthday = 365;
+		
+		public Aging(Person p, int ticksPerDay) {
+			this.target = p;
+			this.ticksUntilNextBirthday = 365 * ticksPerDay;
+		}
+		
+		@Override
+		public void step(SimState arg0) {
+			target.updateAge();
+			arg0.schedule.scheduleOnce(arg0.schedule.getTime() + ticksUntilNextBirthday, this);
+		}
+		
+	}
+	
+	public class Mortality implements Steppable{
+		
+		Person target;
+		int ticksUntilNextMortalityCheck = 0;
+		int tickToCauseMortality = Integer.MAX_VALUE;
+		WorldBankCovid19Sim world;
+		public Mortality( Person p, int ticksPerDay, WorldBankCovid19Sim myWorld ) {
+			this.target = p;
+			this.ticksUntilNextMortalityCheck = 365 * ticksPerDay;
+			this.world = myWorld;
+		} 
+		// steps taken
+		@Override
+		public void step(SimState arg0) {
+			if(this.target.isAlive()) {
+			if (this.tickToCauseMortality < Integer.MAX_VALUE) {
+				causeDeath();
+			}
+			else {
+				
+				determineMortality(arg0);
+			}		
+			}
+		}
+		// functions used
+		private void causeDeath() {
+			if (target.isAlive()) {
+				target.die("");
```

Comment:
it would be good to include an explanation for the default case (ie calling `target.die("<default>")`)